docs/CHANGELOG/changelog-0.1.x.md

```diff
@@ -0,0 +1,20 @@
+# v0.1.0
+
+First official release of Claudie
+
+# Features
+
```

Comment:
this is a nitpick but why is there an empty line after this and the previous heading and not after the others?